credentials/java/com/google/auth/Credentials.java

```diff
@@ -45,6 +45,8 @@ public abstract class Credentials implements Serializable {
 
   public static final String GOOGLE_DEFAULT_UNIVERSE = "googleapis.com";
 
+  private CredentialType credentialType = CredentialType.UNKNOWN;
```

Comment:
Just to confirm: Unknown is default for the other Credential types that aren't currently on our list, right?

Reply:
Yes. And to make it clear, updated this default to "DO_NOT_SEND"